### PR TITLE
task 2 complete

### PR DESCRIPTION
**Pull request by KateCory**

### CODE REVIEW

File: db/seeds/run-seed.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 const devData = require('../data/development-data/index.js');
+const testData = require('../data/test-data')
```
Comment (nc-mezz-davies) [inline db/seeds/run-seed.js:2]: Please put this file back to how it was. You have a before each in your test file that correctly seeds the test DB with the test data. This run seed file is designed to seed the dev DB so it should use the dev data. 😄

File: controllers/topics-controllers.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const { selectTopics } = require("../models/topics-models");
+
+exports.getTopics = (request, response, next) => {
+    selectTopics().then((topics) => {
+        response.status(200).send( { topics });
+
+    });
```
Comment (nc-mezz-davies) [inline controllers/topics-controllers.js:7]: Just missing a catch block here

**Review by nc-mezz-davies — COMMENTED** [review]
Looking good in the main - good testing and nice generic 404 (also with testing). I have left a few comments so please have a look. No need to resubmit but consider doing as part of another task or even on another branch when you have finished a task, that you can just merge and pull straight away

File: db/seeds/run-seed.js (hunk 1 above)
Comment (KateCory) [reply db/seeds/run-seed.js:2]: Ah right yes makes sense! Thanks

File: controllers/topics-controllers.js (hunk 2 above)
Comment (KateCory) [reply controllers/topics-controllers.js:7]: Ah ok thank you

**Pull request merged**